examples/bri-3/src/app.module.ts

```diff
@@ -3,13 +3,13 @@ import { PrismaService } from '../prisma/prisma.service';
 import { CommunicationModule } from './bri/communication/communication.module';
 import { IdentityModule } from './bri/identity/identity.module';
 import { TransactionModule } from './bri/transactions/transactions.module';
-import { WorkgroupModule } from './bri/workgroup/workgroup.module';
+import { WorkgroupsModule } from './bri/workgroup/workgroup.module';
 import { LoggingModule } from './shared/logging/logging.module';
 
 @Module({
   imports: [
     IdentityModule,
-    WorkgroupModule,
+    WorkgroupsModule,
```

Comment:
This parent module name sounds confusing to me since we also have a child workgroup module which deals with the workgroup entity itself. I cannot think of a better name though (Workgrouping, Workorchestration came to my mind). Any suggestions?

Reply:
no strong opinion on this, but i am more for workgroup than trying to introduce new name just to avoid having child with same name, just to avoid new terminology

seems like parent module is Workgroups and child Workgroup which seems like ok distinction

also skimmed the standard sections could not find better name for this, they are part of agreement execution section (https://github.com/eea-oasis/baseline-standard/blob/main/core/baseline-core-v1.0-psd01.md#6-agreement-execution) but transactions are also there so i guess we can not call this module agreement execution

Reply:
@skosito @ognjenkurtic How about WorkgroupManagerModule?

Reply:
i think it is not in line with the other modules. Let's go with what @skosito suggested if you agree, parent plural and child singular.

Reply:
Sticking with parent plural and child singular.